pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -82,18 +83,18 @@ func (p RESTStorageProvider) storage(version schema.GroupVersion, apiResourceCon
 	once := new(sync.Once)
 	var (
 		authorizationRuleResolver  rbacregistryvalidation.AuthorizationRuleResolver
-		rolesStorage               rest.StandardStorage
-		roleBindingsStorage        rest.StandardStorage
-		clusterRolesStorage        rest.StandardStorage
-		clusterRoleBindingsStorage rest.StandardStorage
+		rolesStorage               *genericregistry.Store
+		roleBindingsStorage        *genericregistry.Store
+		clusterRolesStorage        *genericregistry.Store
+		clusterRoleBindingsStorage *genericregistry.Store
 	)
 
 	initializeStorage := func() {
 		once.Do(func() {
-			rolesStorage = rolestore.NewREST(restOptionsGetter)
-			roleBindingsStorage = rolebindingstore.NewREST(restOptionsGetter)
-			clusterRolesStorage = clusterrolestore.NewREST(restOptionsGetter)
-			clusterRoleBindingsStorage = clusterrolebindingstore.NewREST(restOptionsGetter)
+			rolesStorage = rolestore.NewREST(restOptionsGetter).Store
```

Comment:
I'd expect to be able to use the returned value as the storage... shouldn't it satisfy the interface?